src/main/java/com/programmers/ticketparis/controller/performance/PerformanceController.java

```diff
@@ -0,0 +1,52 @@
+package com.programmers.ticketparis.controller.performance;
+
+import com.programmers.ticketparis.dto.performance.request.PerformanceCreateRequest;
+import com.programmers.ticketparis.dto.performance.request.PerformanceUpdateRequest;
+import com.programmers.ticketparis.dto.performance.response.PerformanceResponse;
+import com.programmers.ticketparis.service.performance.PerformanceService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/performances")
+public class PerformanceController {
+    private final PerformanceService performanceService;
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public void createPerformance(@RequestBody PerformanceCreateRequest createRequest) {
+        performanceService.createPerformance(createRequest);
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<PerformanceResponse> findPerformanceById(@PathVariable Long id) {
+        Optional<PerformanceResponse> response = performanceService.findPerformanceById(id);
+        if (response.isPresent()) {
+            return new ResponseEntity<>(response.get(), HttpStatus.OK);
+        } else {
+            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+        }
+    }
+
+    @GetMapping
+    public List<PerformanceResponse> findAllPerformance() {
+        return performanceService.findPerformanceAll();
+    }
+
+    @PatchMapping("/{id}")
+    public ResponseEntity<PerformanceResponse> updatePerformance(@PathVariable Long id, @RequestBody PerformanceUpdateRequest updateRequest) {
+        return new ResponseEntity<>(performanceService.updatePerformance(id, updateRequest), HttpStatus.OK);
```

Comment:
`p4`
ok 정적메소드도 있답니다~~

```suggestion
        return new ResponseEntity.ok(performanceService.updatePerformance(id, updateRequest));
```

Reply:
넵! 이후 팀에서 PeformanceIdResponse를 반환하는 형식으로 의논하여 해당 부분을 수정했습니다.
반영: 3d82dd9e6093837b569e0b149159e502f5276a04, 7c57be96bd439823c4ca968efebb42c302513eb4